stix_shifter_modules/msatp/configuration/lang_en.json

```diff
@@ -37,6 +37,18 @@
                 "label": "Client Secret",
                 "description": "Client Secret of Azure Active directory Application with access to the Microsoft Graph API"
             }
+        },
+        "includeAlerts": {
+            "label": "Join events with Alerts?",
+            "description": "With this feature enabled every event will be searched for correlated alerts in the DeviceAlertsEvent table and the alert data will be included in the results. Keep in mind that this feature will add API CPU consumption."
+        },
+        "includeHostOs": {
+            "label": "Join events with Host OS info?",
+            "description": "With this feature enabled every event will include also the device OS details from the DeviceInfo table. Keep in mind that this feature will add API CPU consumption."
+        },
+        "includeNetworkInfo": {
+            "label": "Join events with Device Network info? ('true' for yes, anything else for no)",
```

Comment:
"Join events with Device Network info? ('true' for yes, anything else for no)" -> "Join events with Device Network info?"

Reply:
good catch